packages/api/test/__fixtures__/sdk/simple-js-cjs/schemas.d.ts

```diff
@@ -1,196 +1,5 @@
-};
-declare const Pet: {
-    readonly type: "object";
-    readonly required: readonly ["name", "photoUrls"];
-    readonly properties: {
-        readonly id: {
-            readonly type: "integer";
-            readonly format: "int64";
-            readonly readOnly: true;
-            readonly default: 40;
-            readonly examples: readonly [25];
-            readonly minimum: -9223372036854776000;
-            readonly maximum: 9223372036854776000;
-        };
-        readonly category: {
-            readonly type: "object";
-            readonly properties: {
-                readonly id: {
-                    readonly type: "integer";
-                    readonly format: "int64";
-                    readonly minimum: -9223372036854776000;
-                    readonly maximum: 9223372036854776000;
-                };
-                readonly name: {
-                    readonly type: "string";
-                };
-            };
-            readonly title: "Category";
-            readonly "x-readme-ref-name": "Category";
-        };
-        readonly name: {
-            readonly type: "string";
-            readonly examples: readonly ["doggie"];
-        };
-        readonly photoUrls: {
-            readonly type: "array";
-            readonly items: {
-                readonly type: "string";
-                readonly examples: readonly ["https://example.com/photo.png"];
-            };
-        };
-        readonly tags: {
-            readonly type: "array";
-            readonly items: {
-                readonly type: "object";
-                readonly properties: {
-                    readonly id: {
-                        readonly type: "integer";
-                        readonly format: "int64";
-                        readonly minimum: -9223372036854776000;
-                        readonly maximum: 9223372036854776000;
-                    };
-                    readonly name: {
-                        readonly type: "string";
-                    };
-                };
-                readonly title: "Tag";
-                readonly "x-readme-ref-name": "Tag";
-            };
-        };
-        readonly status: {
-            readonly type: "string";
-            readonly description: "pet status in the store\n\n`available` `pending` `sold`";
-            readonly enum: readonly ["available", "pending", "sold"];
-        };
-    };
-    readonly title: "Pet";
-    readonly "x-readme-ref-name": "Pet";
-};
-declare const Tag: {
-    readonly type: "object";
-    readonly properties: {
-        readonly id: {
-            readonly type: "integer";
-            readonly format: "int64";
-            readonly minimum: -9223372036854776000;
-            readonly maximum: 9223372036854776000;
-        };
-        readonly name: {
-            readonly type: "string";
-        };
-    };
-    readonly title: "Tag";
-    readonly "x-readme-ref-name": "Tag";
-};
+import Category from './schemas/Category';
```

Comment:
This is wrong because CJS and ESM compilation with `ts-morph` don't create this `schemas/` directory but because we're overhauling the CJS and ESM compilation work with https://github.com/readmeio/api/pull/754 I'm going to leave this breakage alone.